binarytree/__init__.py

```diff
@@ -314,6 +315,38 @@ def _get_tree_properties(root):
     }
 
 
+def _get_parent_node(root, node):
```

Comment:
Instead of using `wtree` (which feels very hacky to be honest), I suggest providing the functionality as a public function with signature `get_parent(root, node)`.

Reply:
What do you think about add the property `up` or I used `upper` for the `Node` object? Still using the function `get_parent(root, node)`.

Reply:
I don't think it can work cleanly as a property since you need access to the root and because we should not be mutating objects. I think a public function might be the only way.